### PR TITLE
Support file operations and upxed PE file on Windows

### LINKED ISSUE #751: api AreFileApisANSI is not implemented
When I testing a PE file compile by VS 2019, qiling show this error, is that mean qiling cant emulate this api not yet, or something wrong when I configure the environment?
Code:
------------------------------------------------
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(int argc, const char** argv)
{
    printf("Hello, World!\n");

    return 0;
}
------------------------------------------------

### PR DESCRIPTION
**Pull request by bet4it**
### Which kind of PR do you create?

- [x] This PR only contains minor fixes.
- [ ] This PR contains major feature update.
- [ ] This PR introduces a new function/api for Qiling Framework.

### Coding convention?

- [x] The new code conforms to Qiling Framework naming convention.
- [x] The imports are arranged properly.
- [x] Essential comments are added.
- [x] The reference of the new code is pointed out.

### Extra tests?

- [ ] No extra tests are needed for this PR.
- [x] I have added enough tests for this PR.
- [ ] Tests will be added after some discussion and review.

### Changelog?

- [ ] This PR doesn't need to update Changelog.
- [x] Changelog will be updated after some proper review.
- [ ] Changelog has been updated in my PR.

### Target branch?

- [x] The target branch is dev branch.

### One last thing

- [x] I have read the [contribution guide](https://docs.qiling.io/en/latest/contribution/)

-----

Security cookie is inited by `___security_init_cookie`, we shouldn't change it by ourself, and this will prevent UPXed file to run.

`file_upx.c` is compiled by cl ( VS 2019 ) and compressed by UPX. The binary is provided in qilingframework/rootfs#26.

Rework of 97cd0b9e57ee45b9ca629fa6fb77011c47c6c0c8
Fix #751
Refer: #452, #836

### CODE REVIEW

**Comment by xwings** [comment]
Hi,

Will you be able to fix the test ?

**Comment by bet4it** [comment]
The test failed because you didn't merge the testcase binary into rootfs🙃

**Comment by xwings** [comment]
> The test failed because you didn't merge the testcase binary into rootfs🙃

Sorry!

Got a busy December and i totally overlooked. merged the rootfs and i will rerun the test.

File: qiling/loader/pe.py [hunk 1]
```diff
@@ -603,19 +608,6 @@ def load(self):
             self.pe.parse_data_directories()
             data = bytearray(self.pe.get_memory_mapped_image())
             self.ql.mem.write(self.pe_image_address, bytes(data))
-            # setup IMAGE_LOAD_CONFIG_DIRECTORY
```
Comment (xwings) [inline qiling/loader/pe.py:606]: the security coocke is not required anymore ?

**Review by kabeor — APPROVED** [review]

File: qiling/loader/pe.py (hunk 1 above)
Comment (xwings) [reply qiling/loader/pe.py:606]: my bad. i saw the comment. 
merge now

**Review by xwings — APPROVED** [review]

**Pull request merged**